scheduler-worker/src/test/java/gov/nasa/jpl/aerie/scheduler/worker/services/SchedulingIntegrationTests.java

```diff
@@ -1144,7 +1339,7 @@ private SchedulingRunResults runScheduler(
   )
   {
     final var activities = new HashMap<ActivityDirectiveId, ActivityDirective>();
-    long id = 0L;
+    long id = 1;
```

Comment:
No idea I'll revert.

Reply:
because if the input acts start at 0, their opposite id will be used in the scheduler and the opposite of 0 is 0, that is the same as the first id that will be used by the scheduler. I am surprised but this shows up only when running the tests individually somehow.